bower.json

```diff
@@ -13,6 +13,6 @@
     "!README.md"
   ],
   "main": [
-    "dist/**/*"
+    "dist/FeatureFlags.js"
```

Comment:
Thats fine with me, but can you make this casing of the filename the same please. I know its only a small thing.